gradle/libs.versions.toml

```diff
@@ -26,6 +28,9 @@ retrofit = {group = "com.squareup.retrofit2", name = "retrofit", version.ref = "
 retrofit-gson = { module = "com.squareup.retrofit2:converter-gson", version.ref = "retrofit" }
 picasso = { group = "com.squareup.picasso", name = "picasso", version.ref = "picasso" }
 androidx-recyclerview = { group = "androidx.recyclerview", name = "recyclerview", version.ref = "recyclerview" }
+viermodel = { group = "androidx.lifecycle", name = "lifecycle-viewmodel-ktx", version.ref = "viewmodel" }
```

Comment:
viermodel?
lifedata?

Reply:
Old test libraries for the implementation of the MainViewModel, I didn't remember to delete them, it will be solved in the next commit.